components/match2/wikis/leagueoflegends/match_group_input_custom.lua

```diff
@@ -585,42 +353,48 @@ function MapFunctions.getParticipants(MatchParser, map, opponents)
 		veto.character = getCharacterName(veto.character)
 	end)
 
-	map.participants = participants
-	map.extradata = extradata
+	Table.mergeInto(extradata, MapFunctions.getAdditionalExtraData(MatchParser, map))
 
-	return MapFunctions.getAdditionalExtraData(MatchParser, map)
+	return participants, extradata
 end
 
 -- Calculate Score and Winner of the map
----@param map table
----@return table
+---@param map {finished:string?, winner:string?}]
+---@return {finished:boolean, winner:integer?, resulttype:string?, walkover:string?, scores:integer[]}
 function MapFunctions.getScoresAndWinner(map)
```

Comment:
map default losses (ff, dq, l) aren't a thing on lol?

Reply:
Not in the live version of the code at least